tests/unit/scripting.tcl

```diff
@@ -146,6 +146,14 @@ start_server {tags {"scripting"}} {
         } 1 x
     } {number 1}
 
+    test {EVAL - Lua number -> Redis integer conversion} {
+        r set x 0
```

Comment:
```suggestion
```

Reply:
Thanks, blind copying from the test above.